target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4019-linksys_ea8300.dts

```diff
@@ -0,0 +1,394 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/*
+ * Device Tree Source for Linksys EA8300 (Dallas)
+ *
+ * Copyright (C) 2019 Jeff Kletsky
+ *
+ */
+
+/dts-v1/;
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "Linksys EA8300 (Dallas)";
+	compatible = "linksys,ea8300", "qcom,ipq4019";
+
+	aliases {
+		led-boot = &led_wps_amber;
+		led-failsafe = &led_wps;
+		led-running = &led_linksys;
+		led-upgrade = &led_world;
+
+		serial0 = &blsp1_uart1;
+	};
+
+
+	leds {
+		compatible = "gpio-leds";
+
+		// Retain node names from running OEM on EA8300
+
+		// Front panel LEDs, top to bottom
+
+		led_plug: diag {
+			label = "ea8300:amber:plug";
+			gpios = <&tlmm 47 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_world: internet {
+			label = "ea8300:amber:world";
+			gpios = <&tlmm 49 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wps: wps {
+			label = "ea8300:white:wps";
+			gpios = <&tlmm 46 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wps_amber: wps_amber {
+			label = "ea8300:amber:wps";
+			gpios = <&tlmm 22 GPIO_ACTIVE_HIGH>;
+			panic-indicator;
+		};
+
+		led_linksys: pwr {
+			label = "ea8300:white:linksys";
+			gpios = <&tlmm 45 GPIO_ACTIVE_HIGH>;
+		};
+
+		// On back panel, above USB socket
+
+		led_usb: usb {
+			label = "ea8300:green:usb";
+			gpios = <&tlmm 0x3d GPIO_ACTIVE_LOW>;
+		};
+	};
+
+
+	gpio_keys {
```

Comment:
Please choose a generic node name. As per entry 5 on the https://openwrt.org/submitting-patches#dts_checklist Otherwise @ynezz will be mad.